### PR TITLE
Acic18 script requirements

### PR DESCRIPTION
**Pull request by jakobzeitler**

### CODE REVIEW

File: experiments/acic18/script.R [hunk 1]
```diff
@@ -3,6 +3,7 @@ set.seed(1)
 rm(list = ls())
 
 library(grf)
+if(packageVersion("grf") >= '0.10.2') {warning("This script requires grf 0.10.2 or higher");}
```
Comment (swager) [inline experiments/acic18/script.R]: I think this should be a '<' rather than '>='

Comment (swager) [inline experiments/acic18/script.R]: style nitpick: could you please put the warning on its own line, indented (and without the semicolon)

File: experiments/acic18/script.R [hunk 2]
```diff
@@ -46,6 +47,9 @@ cf.raw = causal_forest(X, Y, W,
 varimp = variable_importance(cf.raw)
 selected.idx = which(varimp > mean(varimp))
 
+# Currently failing with error:
```
Comment (swager) [inline experiments/acic18/script.R]: if we have the package version check up top, I believe we don't need this comment

File: experiments/acic18/script.R (hunk 1 above)
Comment (jakobzeitler) [reply experiments/acic18/script.R]: I totally agree.

Comment (jakobzeitler) [reply experiments/acic18/script.R]: Sure thing!

File: experiments/acic18/script.R (hunk 2 above)
Comment (jakobzeitler) [reply experiments/acic18/script.R]: Yes, that was just my personal note!

**Comment by jakobzeitler** [comment]
Hope these changes are ok.

**Pull request merged**